src/main/java/utils/Checker.java

```diff
@@ -0,0 +1,40 @@
+package utils;
+
+import models.Advertisement;
+import models.Email;
+
+/**
+ * This class will be used to validate inputs.
+ * 
+ * @author <a href="https://github.com/Trustacean">Edward</a>
+ */
+
+public class Checker {
+
+    public static boolean passwordLengthValidation(String password) {
+        if (password.length() == 8) {
+            return true;
+        }
+        return false;
+    }
+
+    public static boolean containsNonAlphanumeric(String input) {
+        for (char c : input.toCharArray()) {
+            if (!Character.isLetterOrDigit(c))
+                return true;
+        }
+        return false;
+    }
+
+    public static boolean isAdvertisement(Email mail) {
+        return mail instanceof Advertisement;
+    }
+    
+    // possibly for password
+    public static boolean areEqual(String x, String y) {
```

Comment:
This is unecessary. String can be compared using String().equals(String string). Redundant.